client.go

```diff
@@ -1351,6 +1351,13 @@ func validateQueueName(queueName string) error {
 	return nil
 }
 
+// JobListResult is the result of a job list operation. It contains a list of
+// jobs and a cursor for fetching the next page of results.
+type JobListResult struct {
+	Jobs   []*rivertype.JobRow
+	Cursor *JobListCursor
```

Comment:
Not sure it makes sense to output a single `Cursor` field here. There could be a cursor for _any_ of the entries in the `Jobs` list, so with this name it's not clear if it's the start cursor or end cursor.

I doubt there's much use case for the start cursor, and people can always use the `JobListCursorFromJob` function to create one if they want. Maybe let's just name this `LastCursor` to be unambiguous?